SSH TunnelBuilder/Models/ViewController.swift

```diff
@@ -107,6 +125,24 @@ class ViewController: NSViewController, NSComboBoxDataSource {
         tableView.reloadData()
         
     }
+    
+    func checkIcloudAccountStatus() {
+        
+        CKContainer.default().accountStatus { accountStatus, error in
+            if accountStatus == .noAccount {
+                DispatchQueue.main.async {
```

Comment:
Bytt ut med util klasse med ShowAlertBox
self.ShowAlertBox( alertStyle: NSAlert.Style.critical, message: "This app uses iCloud to store your connection settings. Please sign in to iCloud in System Preferences." )

Reply:
Done. And fun was NOT had by all :P